lib/routers/establishment/index.js

```diff
@@ -12,7 +12,7 @@ router.param('establishment', (req, res, next, id) => {
     .then(() => {
       return Establishment.query()
         .findById(id)
-        .eager('authorisations');
+        .eager('[authorisations, roles]');
```

Comment:
Does this also return the associated profiles along with the roles?

Reply:
I'm going to guess yes, since otherwise it wouldn't work.

Reply:
Actually, no.Working on it.